pump/server_test.go

```diff
@@ -709,7 +710,7 @@ func (s *startServerSuite) TestStartPumpServer(c *C) {
 	cfg := &Config{
 		ListenAddr:        "http://127.0.0.1:8250",
 		AdvertiseAddr:     "http://127.0.0.1:8260",
-		Socket:            "unix://127.0.0.1:" + string(time.Now().UnixNano()) + "/hello/world",
+		Socket:            "unix://127.0.0.1:" + strconv.FormatInt(time.Now().UnixNano(), 10) + "/hello/world",
```

Comment:
there's no way a port can bigger than 65536, but this code works, look strange, maybe better change it.